ethereum/ewm/BREthereumEWM.c

```diff
@@ -653,6 +663,27 @@ ewmDestroy (BREthereumEWM ewm) {
     free (ewm);
 }
 
+/// MARK: - Start/Stop
+
+extern void
+ewmStart (BREthereumEWM ewm) {
+    // Start the alarm clock.
+    alarmClockStart(alarmClock);
+
+    // Start the EWM thread
+    eventHandlerStart(ewm->handler);
+}
+
+extern void
+ewmStop (BREthereumEWM ewm) {
+    // Stop the alarm clock
+    alarmClockStop (alarmClock);
+
```

Comment:
Devil's advocate (not sure myself if we want this), but should stop imply disconnect? Almost like how on Android the lifecycle is onCreate->onStart->onResume->onPause->onStop->onDestroy (so, ewmCreate->ewmStart->ewmConnect->ewmDisconnect->ewmStop->ewmDestroy)?

Reply:
Maybe it is worse.... disconnect => stop (which it shouldn't); stop !=> disconnect (which it should).

Reply:
Updated: `ewmStop()` calls `ewmDisconnect()`.  

Added a number of TODOs - for example, noting that start() and stop() don't check on the current state (which they arguably should).